### PR TITLE
Add CenterDistF<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: functions/center_dist.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  void run(BladeBase* blade) {
+    center_.run(blade);
+    center = center_.getInteger(0);
+    num_leds_ = blade->num_leds();
+  }
+  int getInteger(int led) { return abs(led * 32768 / num_leds_ - center); }
+
+private:
+  CENTER center_;
+  int center;
```
Comment (profezzorn) [inline functions/center_dist.h:20]: We should remove the center variable and inline the call to getInteger()

Comment (fdarosa2663) [reply functions/center_dist.h]: updated

File: functions/center_dist.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// CENTER : FUNCTION (defaults to Int<16384>)
+
+template<class CENTER = Int<16384>>
+class CenterDistF {
+public:
+  void run(BladeBase* blade) {
+    center_.run(blade);
+    num_leds_ = blade->num_leds();
+  }
+  int getInteger(int led) { return abs(led * 32768 / num_leds_ - center_.getInteger(0)); }
```
Comment (profezzorn) [inline functions/center_dist.h:15]: It should be getInteger(led) not getInteger(0)

Comment (fdarosa2663) [reply functions/center_dist.h]: Updated

**Pull request merged**